regulations/static/regulations/css/less/layout.less

```diff
@@ -4,6 +4,11 @@
  layout.less defines the structural layout of the site
  */
 
+// focus for accessibliity tabbing
+a:focus {
+    border: 1px dotted @black;
+}
```

Comment:
Looks like there might be a double border as a result:
Production:
<img width="236" alt="screen shot 2016-06-07 at 1 21 40 pm" src="https://cloud.githubusercontent.com/assets/326918/15867807/dce71afa-2cb2-11e6-8eb5-ff2a2e7dbe09.png">
<img width="234" alt="screen shot 2016-06-07 at 1 21 32 pm" src="https://cloud.githubusercontent.com/assets/326918/15867808/dcf16866-2cb2-11e6-8257-a94c66443c2c.png">

Branch:
<img width="230" alt="screen shot 2016-06-07 at 1 21 18 pm" src="https://cloud.githubusercontent.com/assets/326918/15867814/e36bc998-2cb2-11e6-830c-b74a9cd3787a.png">
<img width="235" alt="screen shot 2016-06-07 at 1 21 10 pm" src="https://cloud.githubusercontent.com/assets/326918/15867815/e3718fe0-2cb2-11e6-86cd-fd503cac2040.png">

The extra border is shifting the icon to the left when activating.